charts/bookkeeper-operator/templates/version_map.yaml

```diff
@@ -0,0 +1,18 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: bk-supported-versions-map
+data:
+  keys: |
+        0.1.0:0.1.0
+        0.2.0:0.2.0
+        0.3.0:0.3.0,0.3.1,0.3.2
+        0.3.1:0.3.1,0.3.2
+        0.3.2:0.3.2
+        0.4.0:0.4.0
+        0.5.0:0.5.0,0.5.1,0.6.0,0.6.1,0.6.2,0.7.0
+        0.5.1:0.5.1,0.6.0,0.6.1,0.6.2,0.7.0
+        0.6.0:0.6.0,0.6.1,0.6.2,0.7.0
+        0.6.1:0.6.1,0.6.2,0.7.0
+        0.6.2:0.6.2,0.7.0
+        0.7.0:0.7.0
```

Comment:
Could you please add the 0.7.1 version here?
It's not possible to deploy a 0.7.1 bk image at the moment.

Reply:
@jkhalack We won't be adding versions not yet released on open source.
You can add those to your config map yaml, if needed.
Charts and deploy files provided in p-operator repo are only for reference.

Reply:
@pbelgundi Are you saying that this file is for example purposes only? 
Why is it placed into the operator chart then?

Reply:
@jkhalack Not sure why this should be so difficult to understand .... but charts and deploy files provided in the p-operator repository are for use by pravega opensource users and aligned to pravega opensource releases. If you choose to use different image versions of pravega you obviously would need to modify the .yaml to add those versions. That was the sole reason for bringing the version map outside the code and injecting as a config map.

Reply:
@pbelgundi I don't understand the updated use case for the versions map.
Should it now have a fixed name `bk-supported-versions-map`, not related to the name of BK cluster (as it was assumed previously)?

Reply:
The updated use case is that the name of the map can be anything as long as the same file is mounted to the operator volume. Hence as opposed to earlier where we needed the name to be in a certain format, now we don't have any such requirement for the name of this configmap.

Reply:
I meant the following block:
```
metadata:
  name: bk-supported-versions-map
```
Can it be anything?

Reply:
And why is this conversation resolved if there is no answer to the original question?

Reply:
@SrishT Are you saying that this `metaname.name` for configmap should be the same as line 44 of the operator chart? 
https://github.com/pravega/bookkeeper-operator/blob/master/charts/bookkeeper-operator/templates/operator.yaml#L44

Then we should not need the versions map that used to reside in `charts/bookkeeper/templates` folder. Is this correct?

Reply:
yes Julia that is correct. The version map will not be residing in `charts/bookkeeper/templates` any longer. Also we need to ensure that value in `metaname.name` should be same as the value mentioned in this line https://github.com/pravega/bookkeeper-operator/blob/master/charts/bookkeeper-operator/templates/operator.yaml#L44

Reply:
Thank you, Srishti.
So the situation is that one cannot do any dev work on BK image without either
- downgrading the BK image version tag to `0.7.0`, or
- forking the current repository and making changes to bk-operator chart (note that https://github.com/pravega/bookkeeper-operator/blob/master/charts/bookkeeper-operator/templates/operator.yaml#L44 is not overridable at install time, so either that line has to change or the chart for `bk-supported-versions-map` configmap has to change).

Reply:
I will add `0.7.1` to this map in a PR and get it merged. Does that work @jkhalack